[CK_TILE] Epilogue chaining (Lwpck 3373) (#2773)
ROCm/composable_kernel: include/ck_tile/ops/epilogue/chainer/README.md

# CK Tile Epilogue Chainer

## Overview

> The Epilogue Chainer provides composability for building epilogue sequences for GEMM operations. It allows to chain multiple epilogue stages to be executed sequentially.

Reviewer: ```suggestion
The Epilogue Chainer provides composability for building epilogue sequences for GEMM operations. It provides a means to chain multiple epilogue stages so they can be run sequentially.
```

I switched "executed" to "run" because "execute" is one of the "words to avoid".

## Components

### EpilogueChainer

> EpilogueChainer class, orchestrator that executes epilogue sequences. Takes an initialization epilogue and a sequence of operations to perform.

Reviewer: ```suggestion
The `EpilogueChainer` class is an orchestrator that runs epilogue sequences. It takes an initialization epilogue and a sequence of operations.
```

### EpilogueNode

> Wrapper for individual epilogue operations with embedded arguments. Supports both parameterized and parameter-free epilogues.

Reviewer: ```suggestion
`EpilogueNode` is a wrapper for individual epilogue operations with embedded arguments. Supports both parameterized and parameter-free epilogues.
```

### EpilogueLoop

> Executes a sequence of epilogue nodes across multiple iterations (e.g., for tiled processing).

Reviewer: ```suggestion
`EpilogueLoop runs a sequence of epilogue nodes across multiple iterations (e.g., for tiled processing).
```

Is this a function? So far we have a class and a wrapper, and it makes sense to say what this is.
